packages/extensions-sdk/src/cli/commands/validate.ts

```diff
@@ -0,0 +1,102 @@
+import { EXTENSION_PKG_KEY, EXTENSION_TYPES } from '@directus/extensions';
+import chalk from 'chalk';
+import fse from 'fs-extra';
+import ora from 'ora';
+import { log } from '../utils/logger.js';
+
+export default async function validate(): Promise<void> {
+	const spinner = ora(chalk.bold('Validating Directus extension...')).start();
+
+	try {
+		await Promise.all([checkBuiltCode(), checkReadMe(), checkLicense(), checkDirectusConfig()]);
+
+		spinner.succeed(chalk.bold('Extension is valid'));
+	} catch (error) {
+		spinner.fail(chalk.bold('Failed validation'));
+		log(String(error), 'error');
+	}
+}
+
+async function checkBuiltCode() {
+	const spinner = ora('Check for /dist').start();
+
+	if (!(await fse.pathExists(`${process.cwd()}/dist`))) {
+		spinner.fail();
+		return Promise.reject('No /dist directory');
```

Comment:
As this is an async awaited function we can throw an error instead of returning a rejected promise. Same for most/all other occurrences.
```suggestion
		throw new Error('No /dist directory');
```